packages/lesswrong/lib/collections/comments/fragments.ts

```diff
@@ -74,6 +74,13 @@ registerFragment(`
     rejectedReason
     modGPTRecommendation
     originalDialogueId
+    doppelComments {
+      ...DoppelCommentsFragment
```

Comment:
As Jim noted... I am worried about performance impact, both in the "first time a post is loaded" case and otherwise.

In the first-time case, any idea how many tokens we should expect it to eat (i.e. per comment), any risk of getting rate-limited, etc?